src/test/java/selenium/webdriver/DownloadScriptTest.java

```diff
@@ -0,0 +1,56 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static java.time.Duration.ofSeconds;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class DownloadScriptTest {
+    private WebDriver driver;
+
+    @BeforeEach
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-ssl-errors=yes", "--ignore-certificate-errors");
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    @DisplayName("Download and refresh the page when percentage is >= 50")
+    public void refreshPage() {
+        driver.get("https://demo.seleniumeasy.com/bootstrap-download-progress-demo.html");
+        driver.findElement(By.id("cricle-btn")).click();
+        assertTrue(IsPercentGraterThenFifty());
```

Comment:
please provide the 2nd parameter with error message here

Reply:
Done